code/Common/Importer.cpp

```diff
@@ -78,6 +78,7 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #include <assimp/Profiler.h>
 #include <assimp/commonMetaData.h>
 
+#include <exception>
```

Comment:
GCC 11 has reorganized the stdlib header dependencies, so std::exception_ptr no longer comes in transitively, need to add these.